kimetaro.py

```diff
@@ -128,17 +119,32 @@ def add(message):
         Expected pattern:`/add string1 string2`
         '''
         for v in item:
-            LIST[message.channel.id].append(v)
-            added_list.append(v)
+            if canAppend(message):
```

Comment:
This if/else condition is duplicated.
Could you make it method?
i.e. add_item()

Reply:
Hey, I updated.
Could you please review and merge?